main/survey/survey.lib.php

```diff
@@ -5189,4 +5323,62 @@ static function show_link_available($user_id, $survey_code, $user_answer)
             return false;
         }
     }
+
+    /**
+     * Display survey question chart
+     * @param	array	Chart data
+     * @param	boolean	Tells if the chart has a serie. False by default
+     * @return	void 	(direct output)
+     */
+    static function drawChart($chartData, $hasSerie = false)
+    {
+        $htmlChart = '<div id="chartContainer" class="span12">'.
```

Comment:
Also... SVG is not supported by IE8 (which we still support with Chamilo). Isn't this Dimple thing going to generate SVG-only?
I agree that the code looks a bit too "generated by hand" to feel safe.